### PR TITLE
Feature/unzipper lambda function

### PR DESCRIPTION
**Pull request by clayton-duarte**
Includes:
- CloudFormation configs
- IAM permissions
- S3 Bucket
- S3 Lambda trigger
- Lambda to do the job!

Excludes:
- Tests
- Logs

Notes:
Looks like CloudFormation does not support the use of "external" S3 Buckets. It happens because it doesn't expect only the bucked name, but an instance of a `S3.Bucket` class to connect the S3 event to a Lambda execussion. We can try to figure out how to do it, or, just ditch the old bucket DevOps created for us and use the one created through CloudFormation.

### CODE REVIEW

File: lambdas/index.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { S3Handler } from 'aws-lambda'
+import * as unzipper from 'unzipper'
+import * as AWS from 'aws-sdk'
+
+const S3 = new AWS.S3()
+
+const BUCKET_NAME = process.env.BUCKET_NAME || ''
```
Comment (ohsabry) [inline lambdas/index.ts:7]: probably want to crash and log if process.env.BUCKET_NAME is `undefined`

File: lambdas/index.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      } else {
+        // if not, remove from stream
+        // https://github.com/ZJONSSON/node-unzipper#parse-zip-file-contents
+        entry.autodrain()
+      }
+    }
+
+    await Promise.all(uploadPromises)
+  } catch (error) {
+    // TODO: log to cloudwatch?
```
Comment (ohsabry) [inline lambdas/index.ts:45]: Interesting - can you make a ticket for us to figure out how to alert and monitor these lambdas?

File: lambdas/index.ts (hunk 1 above)
Comment (clayton-duarte) [reply lambdas/index.ts:7]: agreed! BTW, the BUCKET_NAME var comes from the `environment` passed to the `LambdaNodeJs.NodejsFunction` instance, not from the server. But yes, if we forget to pass it inside the code we should throw

File: lambdas/index.ts (hunk 2 above)
Comment (clayton-duarte) [reply lambdas/index.ts:45]: Using CloudWatch I know how, but using Data Dog/Sentry would be much better. Creating a ticket for it!

**Pull request closed without merging**